### PR TITLE
bump(*): Remove origin dependency

### PR DESCRIPTION
**Pull request by adambkaplan**
glide.yaml also reordered to conform to rest of org's conventions.

### CODE REVIEW

File: pkg/build/builder/sti.go [hunk 1]
```diff
@@ -335,7 +334,7 @@ func (s *S2IBuilder) Build() error {
 			)
 		} else {
 			s.build.Status.Reason = buildapiv1.StatusReasonGenericBuildFailed
-			s.build.Status.Message = build.StatusMessageGenericBuildFailed
+			s.build.Status.Message = "Generic Build failure - check logs for details."
```
Comment (adambkaplan) [inline pkg/build/builder/sti.go]: 1.5 million lines of code for _this_?

**Comment by coreydaley** [comment]
/lgtm

looks like we are having some aws infrastructure provisioning issues.

**Pull request merged**